integration/integration.go

```diff
@@ -113,9 +113,9 @@ func testMySQLWithCheckingOldvValue() {
 
 func runTests(cases []framework.Task, env framework.Environment) {
 	log.SetLevel(zapcore.DebugLevel)
-	env.Setup()
 
 	for i := range cases {
+		env.Setup()
```

Comment:
Some behaviors duplicated in `env.Reset` and `env.Setup`, and part of them are not idempotent, such as `createConnector` in avro test. I have changed `createConnector` to idempotent (ignore the 409 error returned from JdbcSinkConnector

Reply:
Is there any special reason to change the location of `env.Setup()` @overvenus

Reply:
`env.Reset` resets all TiCDC data in PD, cli may think TiCDC cluster is not up, so we need `env.Setup` makes sure TiCDC cluster is ready to serve.